torch/nn/functional.py

```diff
@@ -16,6 +16,43 @@
 from . import grad
 
 
+class Reduction:
+    @staticmethod
+    def to_enum(reduction):
```

Comment:
It seems unnecessary to split `to_enum` and `get_reduction_constant` into two methods. The functionals will be called a lot (if used), so let's reduce the python code as much as possible by using one method that takes in all three args.

Reply:
I actually need both of these methods. Sometimes I want to turn the old args into the new strings, for example when I'm calling functional from loss.py. Other times I just want to turn old args into the enums.